invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

```diff
@@ -406,7 +406,17 @@
     "additionalProperties": false,
     "properties": {
       "user": {
-        "type": "integer"
+        "type": ["string", "integer"]
+      }
+    }
+  },
+  "removal_reason": {
+    "type": "object",
+    "additionalProperties": false,
+    "description": "Reason for the removal of a record.",
+    "properties": {
+      "id": {
+        "$ref": "local://definitions-v1.0.0.json#/identifier"
```

Comment:
Hm, yes, that's probably a good idea

Reply:
update: it looks like i missed the prefix here...
it actually references `definitions-v1.0.0.json` (which is defined in [Records-Resources](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/records/jsonschemas/definitions-v1.0.0.json)), rather than `records/definitions-v1.0.0`, which would be the one defined here.

It seems like there's some duplication going on, like the `internal-pid` in [`definitions-v2.0.0`](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/records/jsonschemas/definitions-v2.0.0.json#L139-L163) being the same as in [`records/definitions-v1.2.0`](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/records/jsonschemas/records/definitions-v1.2.0.json#L356-L389) (it has been removed from [`records/definitions-v2.0.0`](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json) though)